src/frontend/utils/validators.ts

```diff
@@ -48,10 +49,10 @@ export const erGyldigMånedDato = (
 };
 
 const barnsVilkårErMellom0og18År = (fom: string, person: IGrunnlagPerson, tom?: string) => {
-    const fødselsdato = familieDayjs(person.fødselsdato);
-    const fødselsdatoPluss18 = familieDayjs(person.fødselsdato).add(18, 'year');
-    const fomDato = familieDayjs(fom);
-    const tomDato = tom ? familieDayjs(tom) : undefined;
+    const fødselsdato = familieDayjs(new Date(person.fødselsdato));
```

Comment:
Hvorfor gjør vi om alt til Date her?

Reply:
For å få satt riktig dato.
familieDayjs('2018-06-12') gir: 2018-06-11T22:00:00.000Z
familieDayjs('2018-06-12', 'YYYY-MM-DD') gir: 2018-06-11T22:00:00.000Z
familieDayjs(new Date('2018-06-12')) gir: 2018-06-12T00:00:00.000Z

Reply:
Så det er noe tiddsone-moro her vi må se mer på. Men for nå ønsket jeg å få rettet denne bugen som det haster litt å få ut :-) Jeg har ikke turt å endre på hele familieDayjs ettersom den brukes overalt.

Reply:
Ok, skjønner! :)